Gen Expressie

----
## Introduction
As of recently, the Institute of Life Sciences (ILC) can perform long-read-sequencing using a MinION sequencer. The MinION sequencer is a small and portable device capable of performing DNA sequencing. DNA sequencing, the process of reading part or all of the DNA of an organism, can be done in both a long-read method or short-read method. Long read sequencing has certain advantages and disadvantages over short-read-sequencing. The pro of using long-read-sequencing is, obvious by the name, that it's able to read longer seqeunces. A downside to long-read sequencing is that the accuracy per read can be much lower than that of short-read sequencing. The ILC has used the MinION for various reasons, such as getting an idea of gene expression by performing RNA sequencing on human cells. Recently, RNA sequencing has been used to generate RNA sequencing data for MCF7 cells. 

MCF7 cells are commonly used in studies that focus on the development of cancer treatment. Due to the number of variants available, it has applications in development of chemotherapeutic drugs and understanding drug resistance. Since the usage of the MinION sequencer is relatively new the ILC wants to know whether the data they are generating is of good quality. In order to accomplish this the ILC wants to compare their data to published data about previous research surrounding MCF7 cells and their expression. A study has been found which has performed RNA sequencing using a MinION sequencer on MCF7 cells and has made their data public @chenSystematicBenchmarkNanopore2021. 

## Endgoal
In order to make the comparison reproducible for future data, an R package needs to be written that follows the exact method of data manipulation used in the paper. To start, we received the fastq-files as a result of the sequencing. These fastq-files will be used as input for a Rmarkdown file, containing bash scripts that will not only check the quality of these files, but also convert them into .BAM files. The quality check is an important start of the process, we obvously want to check if the sequencing gave us good enough results before we even think about visualising the expression. If the quality is deemed good enough, the .BAM files will be used as input for the functions that will be in our R package. First of all, they will be aligned with a reference genome using [Minimap2](https://github.com/lh3/minimap2). After they've been aligned, [Bambu](https://github.com/GoekeLab/bambu) will be used for transcript quantification. Finally, differential gene expression will be analysed by [DESeq2](https://github.com/mikelove/DESeq2). After successfully processing the generated data through the pipeline the quality of the results from the ILC can be compared to that of the paper, to get an idea of the quality.

Between all these steps (and at the end) the data will be visualised in graphs such as scatterplots, heatmaps etc.This will all be sent to our tutor, Chris van Oevelen.

